lib/script.js

```diff
@@ -28,14 +27,15 @@ module.exports = class CovScript {
   }
   _buildLines (source, lines, shebangLength) {
     let position = 0
-    const separator = isWindows ? '\r\n' : '\n'
-    ;(source.trim()).split(separator).forEach((lineStr, i) => {
-      this.eof = position + lineStr.length
+    for (const [i, lineStr] of source.trim().split(/(?<=\r?\n)/u).entries()) {
```

Comment:
Even if a file contains both LF and CR+LF, `/(?<=\r?\n)/u` splits contents as expected.

```javascript
'First\nSecond\r\nThird'.split(/(?<=\r?\n)/u)
//=> ['First\n', 'Second\r\n', 'Third']
```

Reply:
As this library doesn't support legacy Node.js versions, we can safely use lookbehind assertions. https://node.green/#ES2018-features--RegExp-Lookbehind-Assertions